test/test_timeseries.py

```diff
@@ -0,0 +1,5243 @@
+import copy
+import math
+import os
+import statistics as stat
+import sys
+import warnings
+from datetime import timedelta
+from typing import List, Union, cast
+
+sys.path.insert(0, "U:\\Devl\\git\hec-python-library")
```

Comment:
Could this computed from a relative path instead of hardcoded.

Reply:
Added issue to correct.